File: 2026-08/src/day_26/prms_120842/Solution.java
package day_26.prms_120842;

class Solution {
    public int[][] solution(int[] num_list, int n) {
        int[][] arr = new int[num_list.length / n][n];
        for (int i = 0; i < num_list.length / n; i++) {
            System.arraycopy(num_list, i * n, arr[i], 0, n);
        }

        return arr;
    }
}

File: 2026-08/src/day_26/prms_120842/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<vector<int>> solution(vector<int> num_list, int n) {
    vector<vector<int>> v;
    for (int i = 0; i < num_list.size(); i += n) {
        v.emplace_back(num_list.begin() + i, num_list.begin() + i + n);
    }

    return v;
}
